.env.example

```diff
@@ -1 +1,3 @@
+MAINNET_RPC_URL=
 BASE_RPC_URL=
+PRIVATE_KEY=
```

Comment:
Does this private key drive what key is used for the deployer script? Is that a foundry default?

Reply:
Yes, it'll be the deployer's private key. I'm not sure what is foundry default but thought putting the private key in `.env` was better than passing it in as an argument when running `forge script` which we can also do.

Reply:
Right, but this won't work (see feedback on README). I think for now we can just remove it.